openassessment/xblock/test/test_staff_info.py

```diff
@@ -349,6 +350,98 @@ def test_staff_debug_student_info_full_workflow(self, xblock):
         resp = xblock.render_student_info(request)
         self.assertIn("bob answer", resp.body.lower())
 
+    @patch('openassessment.assessment.api.peer.get_data_for_override_score')
+    @scenario('data/example_based_assessment.xml', user_id='Bob')
+    def test_staff_debug_student_info_override_no_score(self, xblock, mock_score_data):
+
+        mock_score_data.return_value = {
+            'points_possible': 10,
+        }
+
+        # Simulate that we are course staff
```

Comment:
It seems like from here to line 371 is repeated across all of these tests, perhaps it can be factored out into a helper?